Assets/Scripts/Game/UserInterfaceWindows/DaggerfallInventoryWindow.cs

```diff
@@ -1065,6 +1036,22 @@ void ShowWagon(bool show)
             Refresh(false);
         }
 
+        private void CheckWagonAccess()
+        {
+            if (allowDungeonWagonAccess)
+            {
+                ShowWagon(true);
+                SelectActionMode(ActionModes.Remove);
+            }
+            else if (GameManager.Instance.PlayerEnterExit.IsPlayerInsideDungeon &&
+                     PlayerEntity.Items.Contains(ItemGroups.Transportation, (int)Transportation.Small_cart) &&
+                     DungeonWagonAccessProximityCheck())
+            {
+                allowDungeonWagonAccess = true;
+                ShowWagon(true);
```

Comment:
It seems it should be
                 allowDungeonWagonAccess = true;
                if (lootTarget == null)
                    ShowWagon(true);
to allow wagon access by proximity but not autoselect it when you loot a corpose while near the entrance
Forums:
  https://forums.dfworkshop.net/viewtopic.php?f=5&t=2728

Reply:
Yeah, I already noticed and fixed it in #1502. Unfortunately it missed the build. Sorry about that.

Reply:
Ah right, it's still not merged
Okay!